swarm/network/messages.go

```diff
@@ -236,14 +236,16 @@ func (self peersMsgData) setTimeout(t *time.Time) {
 	}
 }
 
-func (self peersMsgData) getTimeout() (t *time.Time) {
+/* unused
+func (self *peersMsgData) getTimeout() (t *time.Time) {
```

Comment:
Currently peerMsgData.Timeout is not being used everywhere, so setTimeout could theoretically be removed as well. But, is `peerMsgData.Timeout` some remnant of old code, or preemptive change for future?